silkworm/silkrpc/core/evm_trace.cpp

```diff
@@ -1193,12 +1193,11 @@ awaitable<std::vector<Trace>> TraceCallExecutor::trace_block(const BlockWithHash
         if (!cc) {
             throw std::runtime_error("Invalid chain config");
         }
-        const auto block_rewards = protocol::EthashRuleSet::compute_reward(*cc, block_with_hash.block);
 
         RewardAction action;
         action.author = block_with_hash.block.header.beneficiary;
         action.reward_type = "block";
-        action.value = block_rewards.miner;
```

Comment:
same as above

Reply:
Now written in a way that it will only set reward if ethash is used.